pkg/controller/kubedirectorcluster/members.go

```diff
@@ -239,6 +240,48 @@ func syncMemberNotifies(
 	wgReady.Wait()
 }
 
+// EZML-960
```

Comment:
Note that EZML number (or similar tracking info) doesn't have any meaning in the open-source KD repo here.

Reply:
Replaced by the issue link